stage_descriptions/multiple-matches-03-cy4.md

```diff
@@ -0,0 +1,37 @@
+In this stage, you'll add support for displaying the line numbers of the matching lines.
+
+### The `-n` option
+
+The `-n` option, also known as the `line number` option, is used to display the line number of each matching line.
+
+Example usage:
+
+```bash
+$ echo -ne "dogs\ncat\ndog" | grep -n -E "dogs?"
```

Comment:
Let's ignore this stage - isn't a commonly used flag. And it also doesn't really help with multiple matches, since even if a line had multiple matches the line would be printed only once.